charts/_templates.gotmpl

```diff
@@ -50,6 +50,27 @@ As a result, please:
 
 {{- end }}
 
+{{- define "rstudio.best-practices" }}
+
+## Best Practices
+
+Helm charts are very useful tools for deploying resources into Kubernetes, however, they do require
+some familiarity with kubernetes and `helm` itself. Please ensure you have adequate training and
+IT support before deploying these charts into production environments. Reach out to your account representative
+if you need help deciding whether helm is a good choice for your deployment.
```

Comment:
I would cut all of this; I happened to be browsing the argo cd docs yesterday and I like the "declaration of pre-requisites and some links" pattern https://argo-cd.readthedocs.io/en/stable/user-guide/

Reply:
Should we link to our kubernetes docs from here?

Reply:
I definitely agree - I created an issue linked with this one to track creating such "pre-requisites and links" on the docs.rstudio.com site. Once helm is more mainstream and we have such a resource to link to, I agree that this should go away.